.github/workflows/build_pull_request.yml

```diff
@@ -68,5 +73,7 @@ jobs:
           cache-read-only: true
 
       - name: Build extensions (${{ matrix.chunk.number }})
+        env:
+          MODULES: ${{ join(matrix.chunk.modules, ' ') }}
         run: |
-          ./gradlew $(echo '${{ toJson(matrix.chunk.modules) }}' | jq -r 'join(" ")')
+          ./gradlew $MODULES
```

Comment:
Shellcheck would usually complain about this, but I guess it's intentional here - spaces separate arguments.